src/test/ui/proc-macro/cfg-eval-inner.stdout

```diff
@@ -0,0 +1,310 @@
+PRINT-ATTR INPUT (DISPLAY): impl Foo <
+[u8 ;
+ {
+     # ! [cfg_attr(not(FALSE), rustc_dummy(cursed_inner))] # ! [allow(unused)]
```

Comment:
Hmmm, why does this `cfg_attr` appear in post-`cfg_eval` tokens?
Inner `#![cfg_eval]` on `impl Foo` is supposed to apply to the whole `impl Foo` including generics.

Reply:
This was fixed by switching to the reparse-for-derive approach.